src/main/js/helpers/axis.js

```diff
@@ -714,9 +701,8 @@ const isXAxisOrientationTop = (xAxisOrientation) =>
  */
 const calculateAxesSize = (config) => {
     config.axisSizes = {};
-    config.axisSizes.y = config.padding.hasCustomPadding
-        ? getYAxisWidth(constants.Y_AXIS, config)
-        : getYAxisWidth(constants.Y_AXIS, config) + config.padding.left;
+    config.axisSizes.y =
```

Comment:
@Dinesh94Singh Why did we need the `hasCustomPadding` flag? I am looking for some background. 
Can we remove it altogether from all the constructs (Graph, Gantt and Timeline)?

Reply:
it is to determine if the customPadding is given as input.

inside getPadding function, we return the padding object from constants or default few of the fields to constants when not all fields are given as part of custom padding.

Since config.padding is assigned to the return value of getPadding, there is no way to distinguish if the padding is a custom padding or constant padding.